packages/bitcore-wallet-client/src/lib/common/utils.ts

```diff
@@ -269,6 +269,13 @@ export class Utils {
           );
         }
         break;
+      case Constants.SCRIPT_TYPES.P2TR:
```

Comment:
In the P2PKH section above, you check state on publicKeys. Is that a behavior check which may be useful for all switch cases (and thus, moved above the switch statement), or are there cases where the first arg (in this case, anyway) can be undefined?